### ISSUE TITLE
ACE3 charge range to RDS artillery

### ISSUE THREAD

**Issue by ruPaladin** [acemod/ACE3#1901]
I like the ability in ACE3 mk6mortar to show charge range in first person. Have tried to make the same for RDS mortars and artillery. May you please check it on errors?
Link: https://mega.co.nz/#!yB8wiAjZ!EaUMvuqCpitaH8Bem1YnfDaL19kYdUeYUhAPHXDye14
May it cause FPS drop?

P. S. Based on ACE 3.1.1 ace_mk6mortar.pbo. Require ACE 3.1.1 and RDS static weapon pack.

**Comment by PabstMirror** [comment]
@ruPaladin  - Code looks fine,
I moved away from the `ctrlCreate` as it wasn't always reliably cleaning up (restarting mission in editor)
The cleanest way would probably be to create your own display overlay and put it on that.

I could look into making a more generic version of showing the "charge" level of the current firing mode, that would work on any addon.

Also, with the mk6, the "charge" levels happened to line up perfectly with the index of the firing modes array, that isn't guaranteed to be true on RDS or any other mod, if this isn't the case you would want to look at the `artilleryCharge` in the config and base it on that.

FPS - It does a little code each frame, but really shouldn't have much effect.

Bad news is I just tested the RDS M252 and it looks like it's elevation display isn't accurate, I think it's the same problem as the mk6 had where BIS is doing some magic to level the guns.  So I don't know how useful this will be.
